docs/index.md

```diff
@@ -17,7 +17,7 @@ PM > Install-Package Arcus.EventGrid.All
 - Support for publishing events to an Event Grid Topic ([docs](features/publishing-events))
 - Support for handling endpoint validation handshake ([docs](features/endpoint-validation))
 - Provides tooling for writing integration tests ([docs](features/running-integration-tests))
-- Support for deserializing custom & official Azure events ([docs](features/deserializing-events))
+- Support for deserializing EventGrid, CloudEvent, custom & official Azure events ([docs](features/deserializing-events))
```

Comment:
This is more clear I think - First 2 are about the schema of data while the latter about the envelope
```suggestion
- Support for deserializing official Azure or custom events according to Event Grid & CloudEvent schema ([docs](features/deserializing-events))
```